asab/web/auth/service.py

```diff
@@ -256,6 +256,18 @@ def has_resource_access(self, authorized_resources: typing.Iterable, required_re
 		return True
 
 
+	def has_tenant_access(self, authorized_resources: typing.Iterable, authorized_tenants: typing.Iterable, tenant: str) -> bool:
+		"""
+		Check if the request is authorized to access a tenant.
+		"""
+		if self.Mode == AuthMode.DISABLED:
+			return True
+		if self.has_superuser_access(authorized_resources):
+			return True
```

Comment:
Will it still allow me to invent a tenant and do further stuff with it if I am a superuser?

Reply:
yes, superuser can pass every access check and there is no check if the tenant actually exists